### PR TITLE
Revised `Application`. Fixes #195.

### LINKED ISSUE #195: Revise Application

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

**Comment by m1cm1c** [comment]
Only blanks have been inserted.

**Comment by m1cm1c** [comment]
[PDF](http://m1cm1c.github.io/Beagle/branches/revised-application/Requirements%20Specification.pdf)

**Comment by flyfl** [comment]
lgtm

File: doc/Requirements Specification/Parts/Application.lyx [hunk 1]
```diff
@@ -318,7 +318,8 @@ gls{Beagle}
 
 \end_inset
 
-in order to predict their performance in interaction with the software system.
+ in order to predict their performance in interaction with the software
+ system.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Application.lyx]: „software system“ -> `\glsuserv{system}`

Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Application.lyx]: Thank you.

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**